staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/defaults.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internalversion
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// SetListOptionsDefaults sets defaults on the provided ListOptions if applicable.
+//
+// TODO: once the watch-list fg is always on we register this function in the scheme (via AddTypeDefaultingFunc).
+// TODO: when the function is registered in the scheme remove all callers of this method.
```

Comment:
nit: you can add:
TODO(#115478): ...

which makes tracking all changes in the code easier (even if we forget updating the bug)

Reply:
done